### PR TITLE
Task/27073 search planes, users, requests

### PR DESCRIPTION
**Pull request by phamchien98**
Screencast-from-07-09-2020-10554

### CODE REVIEW

File: app/controllers/planes_controller.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class PlanesController < ApplicationController
   def index
-    @planes = Plane.page(params[:page]).per Settings.pagination.user_page
+    # @planes = Plane.page(params[:page]).per Settings.pagination.user_page
```
Comment (chinhhq-0603) [inline app/controllers/planes_controller.rb]: xóa dòng cmt

**Comment by phamchien98** [comment]
[Redmine](https://edu-redmine.sun-asterisk.vn/issues/27073)

**Comment by phamchien98** [comment]
ready

**Review by chinhhq-0603 — APPROVED** [review]

**Comment by phamchien98** [comment]
ready

**Comment by phamchien98** [comment]
Screenshot from 2020-09-07 16-15-33

**Comment by phamchien98** [comment]
Screenshot from 2020-09-08 09-55-46

**Comment by phamchien98** [comment]
Screenshot from 2020-09-08 10-26-01

**Comment by phamchien98** [comment]
ready

**Comment by phamchien98** [comment]
Screenshot from 2020-09-08 10-45-25

**Comment by phamchien98** [comment]
Screenshot from 2020-09-08 10-45-40

**Comment by phamchien98** [comment]
ready

**Comment by phamchien98** [comment]
Screenshot from 2020-09-08 10-52-40

File: app/assets/stylesheets/requests/index.scss [hunk 2]
```diff
@@ -3,3 +3,30 @@
     font-size: 24px;
   }
 }
+.flex-form{
```
Comment (hieplt-0914) [inline app/assets/stylesheets/requests/index.scss:7]: cách dòng đối với mỗi block

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -3,6 +3,22 @@
   <div id="success">
     <%= render :partial => "shared/flash_toastr" , :locals => { :flash => flash } %>
   </div>
+<div class="card-header">
+<div class="card-tools">
+  <%= search_form_for @q, class: "input-group input-group-sm flex-form" do |f| %>
+    <%= f.label :admin %>
+    <%= f.radio_button :admin_true, 1 %>
```
Comment (hieplt-0914) [inline app/views/users/index.html.erb]: cho vào settings

**Comment by phamchien98** [comment]
ready

**Pull request merged**